lib/helm-freeze.yaml

```diff
@@ -41,9 +41,8 @@ charts:
     version: 0.31.1
     comment: https://github.com/grafana/loki/tree/master/production/helm
   - name: promtail
-    repo_name: loki
-    version: 0.24.0
-    comment: https://github.com/grafana/loki/tree/master/production/helm
+    repo_name: grafana
```

Comment:
weird it changed name.
Not sure if normal?

Reply:
not a big deal if it works in the end. I think Enzo did the change to better match the official repo name: https://grafana.github.io/helm-charts. It's grafana and not Loki